pkg/controller/configmapcainjector/starter.go

```diff
@@ -0,0 +1,59 @@
+package configmapcainjector
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+
+	"crypto/x509"
+	"encoding/pem"
+	"io/ioutil"
+
+	"github.com/golang/glog"
+	servicecertsignerv1alpha1 "github.com/openshift/api/servicecertsigner/v1alpha1"
+	"k8s.io/client-go/informers"
+)
+
+type ConfigMapCABundleInjectorOptions struct {
+	Config *servicecertsignerv1alpha1.ConfigMapCABundleInjectorConfig
+}
+
+func (o *ConfigMapCABundleInjectorOptions) RunConfigMapCABundleInjector(clientConfig *rest.Config, stopCh <-chan struct{}) error {
+	kubeClient, err := kubernetes.NewForConfig(clientConfig)
+	if err != nil {
+		return err
+	}
+	kubeInformers := informers.NewSharedInformerFactory(kubeClient, 2*time.Minute)
+
+	caPem, err := ioutil.ReadFile(o.Config.CABundleFile)
+	if err != nil {
+		return err
+	}
+	block, _ := pem.Decode(caPem)
```

Comment:
According to https://golang.org/pkg/encoding/pem/, `pem.Decode()` only takes and decodes the first PEM blob. Do we want to be injecting the full chain or just the CA? Can we safely assume that the CA cert will always be the first one in the CA bundle file?

Reply:
This step is just to confirm that there is at least one CA pem in the bundle file, we end up injecting the bytes that we read from the file if so.

Reply:
Ah, my bad, should have read it more carefully. Thanks.